components/table/style/index.less

```diff
@@ -358,9 +358,8 @@
 
     &-extra {
       position: absolute;
-      top: 50%;
-      right: 0;
-      transform: translate(100%, -50%);
+      top: 0;
+      right: -10px;
```

Comment:
Any reason to change styles?

Reply:
Using `transform` creates a new stacking context on the element - resulting into the dropdown rendering behind the text of each row.

Another solution would be to keep the previous styles but add `z-index: 1` to it as well. If you prefer that approach, I can change it.

Reply:
`z-index: 1` may cause more side effect, let's follow your way.

BTW, Could you add `cursor` style to dropdown arrow? It seems to be lost in v4.